Cargo.lock

```diff
@@ -1280,6 +1396,15 @@ version = "0.6.25"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "f497285884f3fcff424ffc933e56d7cbca511def0c9831a7f9b5f6153e3cc89b"
 
+[[package]]
```

Comment:
the dependencies for `run-wasm` largely are different from the rest of the project, so we should exclude it from the workspace, like we do with `deno_webgpu`

Reply:
We can do that.
However the downside would be that wasm-bindgen and wasm-bindgen-cli versions could get out of sync.
Because we have the Cargo.lock checked in this would be less likely but still relies on a manual review to catch this problem.

Possibly we could add a --no-serve flag to cargo-run-wasm to allow running run-wasm in CI to ensure the versions are compatible.

Reply:
ok, that's one of the selling points of this PR, so let's keep it here